src/csharp/Microsoft.Spark/Sql/RowPickler.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using Razorvine.Pickle;
+
+namespace Microsoft.Spark.Sql
+{
+    /// <summary>
+    /// Custom pickler for Row objects.
+    /// </summary>
+    internal class RowPickler : IObjectPickler
```

Comment:
Can we move GenericRowPickler here and change the file name to `CustomPicklers.cs`?

Reply:
Changed. Thanks!